src/Dependencies.targets

```diff
@@ -3,7 +3,9 @@
   <PropertyGroup>
     <IncludeSource>True</IncludeSource>
     <IncludeSymbols>True</IncludeSymbols>
-    <SymbolPackageFormat>snupkg</SymbolPackageFormat>
+    <SymbolPackageFormat>snupkg</SymbolPackageFormat>    
+    <!-- Suppress known NuGet package vulnerabilities to unblock build as we track this kind of security issues internally. -->
+    <WarningsNotAsErrors>NU1901;NU1902;NU1903;NU1904</WarningsNotAsErrors>
```

Comment:
To suppress Warning As Error: Package 'Microsoft.NETCore.App' 2.1.0 has a known high severity vulnerability

Reply:
I don't remember this package. Where is it tracked?

Reply:
It's introduced in test projects.https://github.com/Azure/azure-powershell-common/blob/ceb2c410dc5e83a10859db169882eb692a18e891/src/Strategies.Test/Strategies.Test.csproj#L6C1-L6C1

Reply:
The suppress is same as what we did in azure-powershell repo